builtin/aws/ecs/platform.go

```diff
@@ -1057,6 +1148,71 @@ func (p *Platform) resourceAlbListenerCreate(
 	return nil
 }
 
+func (p *Platform) resourceAlbRedirectListenerDestroy(
+	ctx context.Context,
+	sg terminal.StepGroup,
+	sess *session.Session,
+	log hclog.Logger,
+	state *Resource_Alb_RedirectListener,
+) error {
+	if state.Arn == "" {
+		log.Debug("Missing alb redirect listener ARN - it must not have been created successfully. Skipping delete.")
```

Comment:
Probably will be better to re-phrase to:
`There is no alb redirect listener ARN - it was not created successfully. Skipping deletion.`